website/blog/2019-12-09-Own-Your-Data-Pilot.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Pilot project (Tally Zoo) with Own Your Data
+author: Dixon Siu
+---
+
+During MyData 2019 conference (September 2019) in Helsinki, several PDS innovators had come together and discussed openly regarding the interoperability between different PDSs. After the conference, Christoph Fabianek ([Own Your Data](https://www.ownyourdata.eu/)) and Salman Farmanfarmaian ([Freezr](http://www.freezr.info/)) contributed the first draft of the implementation document.
```

Comment:
@dixonsiu If you want to separate summay and detail, use `<!--truncate-->`.

https://raw.githubusercontent.com/facebook/docusaurus/master/website-1.x/blog/2018-12-14-Happy-First-Birthday-Slash.md
https://docusaurus.io/blog/

Reply:
I will revise the content.